packages/apps/human-app/frontend/src/shared/components/ui/page-card/page-card.tsx

```diff
@@ -178,14 +138,14 @@ export function PageCard({
               [breakpoints.mobile]: {
                 display: 'flex',
                 width: '100%',
-                justifyContent: backArrowPath ? 'space-between' : 'flex-end',
+                justifyContent: backNavigation ? 'space-between' : 'flex-end',
                 alignItems: 'center',
               },
             }}
           >
-            {backArrowPath && !hiddenArrowButton ? (
+            {backNavigation && !hideBackButton && (
```

Comment:
1. Let's please rename `hideBackButton` to `showBackButton` with `true` as default
2. `backNavigation && !hideBackButton` - this condition controls arrow button displaying in two places, so we can move it above like:
```
const _showBackButton = backNavigation && showBackButton;
/// later use it like
{_showBackButton && (<IconWrapper....
```